public/locales/de/translation.json

```
@@ -223,5 +223,41 @@
     },
     "terms": {
         "title": "Nutzungsbedingungen"
+    },
+    "not_found": {
+        "title": "Seite nicht gefunden",
+        "description": "Ups! Es scheint, dass dieses Geheimnis sich in Luft aufgelöst hat. Keine Sorge, du kannst ein neues erstellen oder unsere Dokumentation ansehen.",
+        "go_home": "Neues Geheimnis erstellen",
+        "view_docs": "API-Dokumentation ansehen"
+    },
+    "statistics": {
+        "heading": "Plattform-Statistiken",
+        "overview": "Übersicht",
+        "security_features": "Sicherheitsfunktionen",
+        "total_secrets": "Gesamtzahl der Geheimnisse",
+        "active_secrets": "Aktive Geheimnisse",
+        "public_secrets": "Öffentliche Geheimnisse",
+        "total_users": "Gesamtzahl der Benutzer",
+        "total_files": "Gesamtzahl der Dateien",
+        "password_protected": "Passwortgeschützt",
+        "ip_restricted": "IP-beschränkt",
+        "average_views": "Durchschnittliche Aufrufe",
+        "views_per_secret": "Aufrufe pro Geheimnis",
+        "description": "Ein umfassender Überblick über die Nutzung Ihrer Plattform und Sicherheitsfunktionen"
```

```suggestion
        "description": "Ein umfassender Überblick über die Nutzung der Plattform und Sicherheitsfunktionen"
```
Here the AI used formal language. It sounds kinda appropriate in this context - but I'd suggest to circumvent this here and just refer to "the platform" instead of "your platform"